datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1399,6 +1409,84 @@ type Dataset implements EntityWithRelationships & Entity & BrowsableEntity {
     exists: Boolean
 }
 
+type RoleAssociation {
+
+    """
+    The Role entity itself
+    """
+    role: Role!
+
+    """
+    Reference back to the tagged urn for tracking purposes e.g. when sibling nodes are merged together
+    """
+    associatedUrn: String!
+
+}
+
+type Access {
+    roles: [RoleAssociation!]
+}
+
+type Role implements Entity{
+    """
+    A primary key of the Metadata Entity
+    """
+    urn: String!
+
+    """
+    A standard Entity Type
+    """
+    type: EntityType!
+
+    """
+    List of relationships between the source Entity and some destination entities with a given types
+    """
+    relationships(input: RelationshipsInput!): EntityRelationshipsResult
+
+    """
+    Id of the Role
+    """
+    id: String!
+
+    """
+    Role properties to include Request Access Url
+    """
+    properties: RoleProperties!
+
+    """
+    A standard Entity Type
+    """
+    actors: Actor!
+
+
+}
+
+type Actor {
+    provisionedUsers: [CorpUser!]
```

Comment:
Question: Why are we have a RoleProvisionedUser object in the GMS Backend layer but we have a list of resolved users here.

My expectation is to see:

```
users: [RoleUser!]! 


...

type RoleUser {
     // The user attached to the role
      user: CorpUser!
}
```

here.

Reply:
Got it. Done